core/assets/erc20/erc20.go

```diff
@@ -51,11 +44,13 @@ type ETHClient interface {
 	CollateralBridgeAddress() ethcommon.Address
 	CurrentHeight(context.Context) (uint64, error)
 	ConfirmationsRequired() uint64
+	ConfiguredChainID() string
```

Comment:
Why not just `ChainID()` here?

Reply:
Because there is a `ChainID(context.Context)` method already on that client wrapper, and that triggers a call.